Settings.XamlStyler

```diff
@@ -0,0 +1,41 @@
+{

```

Comment:
This is a JSON file, and should have a `.json` extension

Reply:
meh, okay. This was the default name for it, and I hadn't updated all the tooling to manually pass the config path yet. This is the equivalent of `.clang_format` - `xstyler` will look for this file upwards till it finds it if none was provided.